.storybook/config.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import 'prismjs';
+import { configure, setAddon } from '@storybook/react';
+import infoAddon from '@storybook/addon-info';
+import { setOptions } from '@storybook/addon-options';
+
+/**
+ * Internal dependencies
+ */
+import * as element from 'element';
+import './style.scss';
+
+function loadStories() {
+	window.wp = { ...window.wp, element };
```

Comment:
Hmm, are we needing to recreate the window global here for everything the components might depend on? We might need a more scalable approach (eventually?).